spec/models/user_spec.rb

```diff
@@ -57,6 +57,20 @@
     end
   end
 
+  describe "Scopes >" do
+    describe "->alphabetized" do
+      let!(:a_name_user) { create(:user, name: 'Amanda') }
+      let!(:z_name_user) { create(:user, name: 'Zachary') }
```

Comment:
I'm not a gigantic fan of `let!`s, but here this is relatively clean; if there is another canonical way to do this in this repo; please lmk :)